examples/ex08.yml

```diff
@@ -7,14 +7,19 @@ connectors:
     pins: [T, R, S]
     pinlabels: [Dot, Dash, Ground]
     show_pincount: false
+    image: ../resources/stereo-phone-plug-TRS.png
```

Comment:
Please move the `resources/` directory into the `examples/` directory, it shouldn't be in the repo's root.

Reply:
I explained the reason and asked your opinion in https://github.com/formatc1702/WireViz/issues/27#issuecomment-668704773:
> I suggest storing images for embedding in a new resources folder adjacent to examples. Then it is easier to access the same folder also from a tutorial if that might be needed in the future. Do you rather prefer a folder below examples?

Do you think it is OK to access `../examples/resources/image.png` from a tutorial or is it never useful to access the same images from both examples and tutorial YAML files? I can do the change, as long as you understand the implications I'm thinking about.

Reply:
If necessary, I see no harm in having both `examples/resources/` and `tutorial/resources` directories if necessary, to avoid having to do any `../` directory traversing. So my answer is yes, I do prefer a folder inside `examples` (and another one inside `tutorials` if needed)

Reply:
I will do as requested. I have drawn a 4-pin Molex KK 254 connector house that could have been useful in both places, though.
![MolexKK254_4-pin](https://user-images.githubusercontent.com/12657033/90235635-f097f900-de21-11ea-92ae-80cf204ca063.png)
I'm sorry for nagging. I forgot to include this image example in my previous comment. I did draw it as SVG, but my Graphviz installation doesn't accept embedded SVG images. 😞

Reply:
Even still, I think it's OK to have two instances of the tiny image file (one in `examples/resources/` and one in `tutorial/resources/`), if that means fewer `../`s in the path :)